### ISSUE TITLE
Next Updates

### ISSUE THREAD

**Issue by finger563** [Max-Mobility/permobil-client#820]
### SD Wear:
- [x] add serial number / user id to sentry logs for better technical support
- [x] add language control to settings
- [x] wireless updates bug fixes (#824 )
- [x] pull language options from translation files (#831 )
- [x] update how we measure smartdrive battery usage
- [x] local notifications - reminders for tire pressure, smartdrive maintenance, new personal record, etc.
- [x] only notify coast time record later in the day so the value is more likely to be final for the day ( #878)
- [x] Periodically check server for new SD / BT firmware and let users know (#860)
- [x] change power assist timeout alert to have actions: `Dismiss`, `Do not show again` - and save a flag to not show the alert if they press the `Do not show again` action.
- [x] shortcut to watch settings page for configuring time - `startActivity(Intent(Settings.ACTION_DATE_SETTINGS))` (#839)
- [x] add language for new error code (008) (#844)
- [x] bugfix: blank screen in errors (#817)
- [x] prompt user to send settings to smartdrive after changing the settings (#337)

### SD Watchface:
- [x] add serial number / user id to sentry logs for better technical support
- [x] tapping on time area toggles display of time and display of date (just like sd wear app)
- [x] update the sizing / scaling of the layout to accomodate other watch face sizes (e.g. C2+)

### PT Wear:
- [x] add serial number / user id to sentry logs for better technical support
- [x] add language control to settings
- [x] pull language options from translation files (#831)
- [x] local notifications - reminders for repositioning / pressure relief, activity per day, new personal record, etc.
- [x] complication / element should open app and request permissions if the app service is not running / the user has not provided the permissions required (#846 )

**Comment by finger563** [comment]
Sd battery usage update (when percentage is measured)

Setting language update

**Comment by finger563** [comment]
## Not scoped for this release:
### SD Wear:
- [ ] add ability to create and view profiles (main page)
- [ ] show current profile on main power assist screen (color, title, ?)
- [ ] add ability to quickly / easily switch between profiles (e.g. by swiping on the main power assist screen when power assist is off)
- [ ] first run / walkthrough / coachmarks
- [ ] google fit integration for smartdrive distance
- [ ] second button support - deactivate power assist / turn off motor [link to multifunction button documentation](https://developer.android.com/training/wearables/ui/multi-function)
- [ ] crown scroll support
- [ ] detect if there is no phone paired (perhaps if `android.support.wearable.phone.PhoneDeviceType.getPhoneDeviceType(ad.getApplicationContext())` returns `android.support.wearable.phone.PhoneDeviceType.DEVICE_TYPE_ERROR_UNKNOWN`?)
- [ ] if there is no phone paired: `connect to pushtracker mobile app` button shows alert informing the user about factory reset, with a button to take you to the factory reset page of settings?
- [ ] if there is no phone paired: wireless updates has an alert with a button which takes you to the wifi settings page to input a wifi network if it detects that there is no network connection

**NOTE: if there is no phone paired (e.g. in TEST_MODE) then the watch reports that the phone paired is an Android phone, but if you try to send intents to it, it will trigger the `result failed` in the `onResultData()` of WearOSComms. We'll want to add a `ResultFailed` callback which can be registered so the calling application can handle the case that the intent failed.**

### PT Wear:
- [ ] first run / walkthrough / coachmarks
- [ ] google fit integration for pushes / coast time
- [ ] second button support (what would it do?)
- [ ] crown scroll support

### PT Mobile:
- [ ] first run / walkthrough / coachmarks
- [ ] survey questions (pain, fatigue, independence) over time
- [ ] Update design of PT mobile: related issues #524 #539 #733 ; primarily to better indicate how to use the app and what the different configurations do.
- [ ] determine if #441 and #416 are still issues after #820 was closed
- [ ] determine if #382 is still an issue after #820

**Comment by finger563** [comment]
Now that we are ready for #880 and we have completed the items selected for these updates, this issue is closed.

The de-scoped issues will make it back into the next wave of updates.
